Wangscape/AlphaCalculator.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include <vector>
+#include <functional>
+#include <SFML/Config.hpp>
+class AlphaCalculator
```

Comment:
Agreed. Many other files could also benefit from better ordering of `#include`s.